test-interface/src/main/scala/sbt/testing/Event.scala

```diff
@@ -1,41 +1,40 @@
 package sbt.testing
 
-/**
- * An event fired by the test framework during a run.
- */
+/** An event fired by the test framework during a run. */
 trait Event {
 
-  /**
-   * The fully qualified name of a class that can rerun the suite or test
-   * about which an event was fired.
+  /** The fully qualified name of a class that can rerun the suite or test
+   *  about which an event was fired.
```

Comment:
No it doesn't. The whole pass was mostly to fix the line lengths.

The "ScalaDoc comments", as you call them, are just not what we use. We use the same convention as that which is used in the scala/scala repo. I don't know anyone who actually defends the style you mention.